.github/workflows/sync-fork.yml

```diff
@@ -13,4 +13,22 @@ jobs:
     steps:
     - env:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: gh repo sync Windows-on-ARM-Experiments/binutils-gdb -b master
\ No newline at end of file
+      run: gh repo sync Windows-on-ARM-Experiments/binutils-gdb -b master
+  
+  build:
+    name: Build and test latest
+    needs: sync-fork
+    uses: ./.github/workflows/makefile.yml
+    with:
+      branch: master
+  
+  sync-default:
```

Comment:
it looks like it could be one job instead of 3

Reply:
@eukarpov As I remember, reusable workflows can not be used inside of job steps, and only can be used as a job.